### ISSUE TITLE
Display bug in Context Data display panel

### ISSUE THREAD

**Issue by dxdc** [node-red/node-red#2460]
### What are the steps to reproduce?

1. Add this variable to global (memoryOnly). Probably not completely required, but it's easy to reproduce this way.

> E.g., global.set('rainInches', ...);

```js
{"2a":{"LastUpdateTime":1581575491494,"Midnight":2.7,"SinceMidnight":0},"1b":{"LastUpdateTime":1581525383086,"Midnight":10.71,"SinceMidnight":0},"64a":{"LastUpdateTime":1581525383088,"Midnight":0.31,"SinceMidnight":0.32},"129b":{"LastUpdateTime":1581525383088,"Midnight":10.71,"SinceMidnight":0},"3b":{"LastUpdateTime":1581525383089,"Midnight":10.39,"SinceMidnight":0},"24a":{"LastUpdateTime":1581525383090,"Midnight":0.04,"SinceMidnight":0},"8a":{"LastUpdateTime":1581525383091,"Midnight":2.52,"SinceMidnight":0},"36e":{"LastUpdateTime":1581525383092,"Midnight":41.57,"SinceMidnight":0},"4e":{"LastUpdateTime":1581525383093,"Midnight":41.57,"SinceMidnight":0},"130b":{"LastUpdateTime":1581525383094,"Midnight":10.12,"SinceMidnight":0},"144a":{"LastUpdateTime":1581525383095,"Midnight":0.08,"SinceMidnight":0},"128a":{"LastUpdateTime":1581525383096,"Midnight":0.08,"SinceMidnight":0},"96a":{"LastUpdateTime":1581525383097,"Midnight":0.16,"SinceMidnight":0},"38c":{"LastUpdateTime":1581525383098,"Midnight":21.42,"SinceMidnight":0},"28c":{"LastUpdateTime":1581525383099,"Midnight":20.24,"SinceMidnight":0},"16a":{"LastUpdateTime":1581525383101,"Midnight":0.16,"SinceMidnight":0},"12e":{"LastUpdateTime":1581525383102,"Midnight":41.57,"SinceMidnight":0},"176a":{"LastUpdateTime":1581525383106,"Midnight":1.26,"SinceMidnight":0},"32a":{"LastUpdateTime":1581525383107,"Midnight":2.52,"SinceMidnight":0},"18c":{"LastUpdateTime":1581525383109,"Midnight":20.79,"SinceMidnight":0},"72a":{"LastUpdateTime":1581525383110,"Midnight":5.71,"SinceMidnight":0},"192a":{"LastUpdateTime":1581525383111,"Midnight":2.52,"SinceMidnight":0},"40a":{"LastUpdateTime":1581525383112,"Midnight":2.52,"SinceMidnight":0},"112a":{"LastUpdateTime":1581525383113,"Midnight":0.31,"SinceMidnight":0},"160c":{"LastUpdateTime":1581525383114,"Midnight":22.83,"SinceMidnight":0},"80a":{"LastUpdateTime":1581525383115,"Midnight":0.63,"SinceMidnight":0},"11b":{"LastUpdateTime":1581525383116,"Midnight":10.39,"SinceMidnight":0},"132e":{"LastUpdateTime":1581525383117,"Midnight":41.57,"SinceMidnight":0},"136c":{"LastUpdateTime":1581525383118,"Midnight":20.24,"SinceMidnight":0},"140e":{"LastUpdateTime":1581525383119,"Midnight":41.57,"SinceMidnight":0},"104b":{"LastUpdateTime":1581525383120,"Midnight":10.39,"SinceMidnight":0},"168a":{"LastUpdateTime":1581525383122,"Midnight":2.52,"SinceMidnight":0},"6c":{"LastUpdateTime":1581525383123,"Midnight":20.79,"SinceMidnight":0},"22a":{"LastUpdateTime":1581525383124,"Midnight":5.08,"SinceMidnight":0},"20a":{"LastUpdateTime":1581525383125,"Midnight":0.63,"SinceMidnight":0},"146c":{"LastUpdateTime":1581525383131,"Midnight":20.79,"SinceMidnight":0},"200a":{"LastUpdateTime":1581525383132,"Midnight":5.71,"SinceMidnight":0},"224a":{"LastUpdateTime":1581525383134,"Midnight":0.16,"SinceMidnight":0},"134c":{"LastUpdateTime":1581525383135,"Midnight":20.79,"SinceMidnight":0},"196e":{"LastUpdateTime":1581525383136,"Midnight":41.57,"SinceMidnight":0},"100c":{"LastUpdateTime":1581525383138,"Midnight":20.18,"SinceMidnight":0},"48a":{"LastUpdateTime":1581525383139,"Midnight":2.52,"SinceMidnight":0},"68e":{"LastUpdateTime":1581525383146,"Midnight":43.17,"SinceMidnight":0},"56a":{"LastUpdateTime":1581525383147,"Midnight":2.52,"SinceMidnight":0},"88b":{"LastUpdateTime":1581525383149,"Midnight":16.69,"SinceMidnight":0},"97b":{"LastUpdateTime":1581525383150,"Midnight":10.39,"SinceMidnight":0},"208b":{"LastUpdateTime":1581525383152,"Midnight":11.42,"SinceMidnight":0},"52e":{"LastUpdateTime":1581525383153,"Midnight":41.57,"SinceMidnight":0},"148e":{"LastUpdateTime":1581525383158,"Midnight":41.57,"SinceMidnight":0},"33b":{"LastUpdateTime":1581525383159,"Midnight":10.16,"SinceMidnight":0},"17b":{"LastUpdateTime":1581525383161,"Midnight":10.39,"SinceMidnight":0},"113e":{"LastUpdateTime":1581525383162,"Midnight":40.39,"SinceMidnight":0},"152a":{"LastUpdateTime":1581525383164,"Midnight":0.04,"SinceMidnight":0},"50c":{"LastUpdateTime":1581525383166,"Midnight":20.77,"SinceMidnight":0},"98b":{"LastUpdateTime":1581525383170,"Midnight":10.39,"SinceMidnight":0},"5a":{"LastUpdateTime":1581525383172,"Midnight":5.2,"SinceMidnight":0},"34b":{"LastUpdateTime":1581525383174,"Midnight":10.39,"SinceMidnight":0},"162c":{"LastUpdateTime":1581525383176,"Midnight":20.79,"SinceMidnight":0},"2c":{"LastUpdateTime":1581576976804,"Midnight":21.42,"SinceMidnight":0},"164c":{"LastUpdateTime":1581545552511,"Midnight":20.79,"SinceMidnight":0},"48b":{"LastUpdateTime":1581553548965,"Midnight":11.02,"SinceMidnight":0},"32e":{"LastUpdateTime":1581597224839,"Midnight":41.57,"SinceMidnight":0}}
```

2. Refresh Context Data panel

3. Expand a few items and click on the timestamp to show as a date

4. Continue expanding items

### What happens?

All variables are shown as dates (even those which are integers)

<img width="659" alt="Screen Shot 2020-02-13 at 6 54 58 AM" src="https://user-images.githubusercontent.com/7200365/74437837-7bc21300-4e2e-11ea-8c73-82d403deb1fc.png">

### What do you expect to happen?

Only dates should be shown as dates. Other variables should retain their correct formatting.

### Please tell us about your environment:

- [ ] Node-RED version: 1.0.3
- [ ] Node.js version: 10.19.0
- [ ] npm version: 6.13.7
- [ ] Platform/OS: Raspberry Pi
- [ ] Browser: Chrome (Mac) 80.0.3987.100 (Official Build) (64-bit)

**Comment by knolleary** [comment]
Thanks for the issue - always appreciate a nice and clear bug report that makes it easy to recreate. Fix pushed to master - will be in 1.0.4.

**Comment by dxdc** [comment]
awesome... thanks for fixing it!
